index.html

```diff
@@ -96,11 +96,16 @@
   &lt;link rel="preconnect" href="//cdn.example.com" crossorigin&gt;
       </pre>
 
-      <ul>
-        <li>The `crossorigin` <a href="https://html.spec.whatwg.org/multipage/infrastructure.html#cors-settings-attributes">CORS setting attribute</a> is an OPTIONAL attribute that indicates the CORS policy for resources that will be fetched from the specified origin - e.g. when set to "anonymous", the user agent SHOULD initiate a handshake for a connection that can be used to fetch resources with an equivalent policy.</li>
+      <p>To <dfn>initiate a preconnect</dfn>, the user agent MUST run these steps:</p>
+      <ol>
+        <li>Let <var>origin</var> be the origin [[!ORIGIN]] of the value of the `href` attribute.</li>
+        <li>If the `crossorigin` <a href="https://html.spec.whatwg.org/multipage/infrastructure.html#cors-settings-attributes">CORS setting attribute</a> value is set to `use-credentials` and <var>origin</var> is not equal to <a href="https://html.spec.whatwg.org/multipage/browsers.html#origin-2">current Document's origin</a>, let <var>credentials</var> be a boolean value set to `true`. Otherwise, let <var>credentials</var> be a boolean value set to `false`.</li>
```

Comment:
I think it would be better to use the terminology from HTML here. That the attribute is in a certain state.